fix: update 0007 "The process of accepting inbound peers" section (#54)
nervosnetwork/rfcs: rfcs/0007-scoring-system-and-network-security/0007-scoring-system-and-network-security.md

CKB refers to Bitcoin's eviction test and steps are as follows:

1. Consider currently connected inbound peers as `candidate_peers`.
2. Protect peers:

>     1. Delete `N` peers from `candidate_peers` which has the highest score.

Reviewer: Where does the `N` come from? Is the `N` identical in step 1, 2 and 3?

    2. Delete `N` peers from `candidate_peers` which has the lowest ping.
    3. Delete `N` peers from `candidate_peers` which most recently sent us messages.

>     4. Delete `candidate_peers.size / 2` peers from `candidate_peers` which has been connected the longest.

Reviewer: ```suggestion
    4. Delete `candidate_peers.size / 2` peers from `candidate_peers` which have the longest connection time.
```

3. Group `candidate_peers` according to `network group` field.
4. Find out the group which contains the most peers.

> 5. Evict the lowest scored peer from the group; if we couldn't find a peer to evict, the new peer would be refused to connect.

Reviewer: ```suggestion
5. Evict the lowest scored peer from the group found in step 4 if it is not empty. Otherwise, reject the connection from the new peer.
```
